src/pdn/src/straps.cpp

```diff
@@ -68,7 +68,8 @@ Straps::Straps(Grid* grid,
   if (spacing_ == 0 && pitch_ != 0) {
     // spacing not defined, so use pitch / (# of nets)
     spacing_ = pitch_ / getNetCount() - width_;
-    spacing_ = TechLayer::snapToManufacturingGrid(getBlock()->getDataBase()->getTech(), spacing_, true);
+    // round down, later check will fail if spacing does not meet min spacing, but rounding up will cause pitch check to fail.
```

Comment:
When the later check fails do you then snap up?

Reply:
no, if you round up then the requested pitch will not be possible with the new spacing.